cvat/apps/engine/models.py

```diff
@@ -265,6 +266,17 @@ class RemoteFile(models.Model):
     class Meta:
         default_permissions = ()
 
+
+class RelatedFile(models.Model):
```

Comment:
Why do we need the db model? I expect that 3d data will be handled  the same way as regular images. Probably I miss something.

Reply:
We have used the db model RelatedFile in-order to place the contextual image corresponding to the PCD files.
Also in future we can have support for multiple images mapped to a single PCD file.